vars/provision.md

```diff
@@ -0,0 +1,25 @@
+# provision
+
+In order to use it with the default conventions simply add the following to
+your Jenkinsfile.
+For the details of what happens please review the
+[provision](provision.groovy) script.
+
+## Prerequisite
+For dynamic resources you will need a `provision.azure.yaml` file in your repo with the below format.
+```
+resources:
+  queues:
+    - name: claim
+    - name: payment
+    - name: schedule
+```
+
+```
+@Library('defra-library@v-8') _
+
+provision environment: 'dev'
+```
```

Comment:
Don't need this part as we're not calling it direct from a pipeline.

Instead include usage of those signatures below